compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -254,6 +254,7 @@
     INVALID_PATTERN_CLAUSES_IN_MATCH_STMT("invalid.pattern.clauses.in.match.stmt"),
     STATIC_MATCH_ONLY_SUPPORTS_ANYDATA("static.value.match.only.supports.anydata"),
     UNINITIALIZED_VARIABLE("uninitialized.variable"),
+    CONTAINS_UNINITIALIZED_FIELDS("uninitialized.fields"),
```

Comment:
Shall we make this specific? like `uninitialized .object.fields`?

Reply:
Changed.